packages/react-components/src/shared/src/index.ts

```diff
@@ -48,3 +48,6 @@ export * from "./useRefState";
 export * from "./slots";
 export * from "./checkable";
 export * from "./styles";
+
+// Types
+export type { SlotAttribute, ChildrenAttribute, AsAttribute, ForwardedRefAttribute } from "./component-types";
```

Comment:
This is a little bit irritating, but since there is no `export type * from ...`, i tried `export * from ...` but it causes an error. I assume it's because `export * from` a file that contains nothing (after the babel transpilation) isn't working properly